skr/go.mod

```diff
@@ -1,9 +1,12 @@
-module github.com/kyma-project/kyma-watcher/operator
+module github.com/kyma-project/kyma-watcher/skr
 
 go 1.17
 
+replace github.com/kyma-project/kyma-watcher/kcp => ../kcp
```

Comment:
Do we want to commit this override? Probably this is just for testing, but I still wanted to ask

Reply:
Yes we intend to keep it until kcp part is merged.